pl_examples/basic_examples/README.md

```diff
@@ -61,11 +61,57 @@ This script shows you how to extract the data related components into a `Lightni
 
 ```bash
 # CPU
-python image_classifier_5_lightning_datamodule.py
+python mnist_examples/image_classifier_5_lightning_datamodule.py
 
 # GPUs (any number)
-python image_classifier_5_lightning_datamodule.py --trainer.gpus 2
+python mnist_examples/image_classifier_5_lightning_datamodule.py --trainer.gpus 2
 
 # Distributed Data Parallel (DDP)
-python image_classifier_5_lightning_datamodule.py --trainer.gpus 2 --trainer.strategy 'ddp'
+python mnist_examples/image_classifier_5_lightning_datamodule.py --trainer.gpus 2 --trainer.strategy 'ddp'
+```
+
+______________________________________________________________________
+
+#### AutoEncoder
+
+This script shows you to implement an CNN auto-encoder.
+
+```bash
+# CPU
+python autoencoder.py
+
+# GPUs (any number)
+python autoencoder.py --trainer.gpus 2
+
+# Distributed Data Parallel (DDP)
+python autoencoder.py --trainer.gpus 2 --trainer.strategy 'ddp'
+```
+
+______________________________________________________________________
+
+#### Backbone Image Classifier
+
+This script shows you to implement a `LightningModule` as a `System`.
+A System describes a `LightningModule` which takes a single a `torch.nn.Module` which makes
```

Comment:
```suggestion
A system describes a `LightningModule` which takes a single `torch.nn.Module` which makes
```
I don't think we have formalized "System" somewhere. It's not a name, right?

Reply:
Removed. But I think we should actually formalized the term. The community has been using it for a while now.